cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/ReadSegmentRangeCommand.java

```diff
@@ -66,27 +67,6 @@ public void execute() throws Exception {
         output("\nThe segment data has been successfully written into %s", fileName);
     }
 
-    /**
```

Comment:
This seems like is moved to the FileHelper class.

Reply:
Yes this has been moved to the `FileHelper` class.